tests/unit/Entity/EntityTest.php

```diff
@@ -14,21 +14,20 @@
  * @deprecated
  * This test class is to be phased out, and should not be used from outside of the component!
  *
- * @group Wikibase
- * @group WikibaseDataModel
```

Comment:
As said in an other pull request, this is unrelated and should be done in a separate pull request. Personally I prefer to add both `@group Wikibase` and `@group WikibaseDataModel` to all tests.